src/test/resources/pages/InstructorHomeHTMLPersistenceCheck.html

```diff
@@ -330,4 +330,21 @@ <h4 class="modal-title">
     </div>
     
 
-<div class="tooltip fade top in" style="top: 97px; left: 625px; display: block;"><div class="tooltip-arrow"></div><div class="tooltip-inner">Search for student's name</div></div></body></html>
```

Comment:
Odd, why is a tooltip there and why is it suddenly removed now? Does it still pass without this?